packages/paste-website/src/pages/patterns/error-state/index.mdx

```diff
@@ -0,0 +1,585 @@
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+## Usage
+
+### General
+
+Use an error state after a problem occurs. For example, a user tried to submit a form with required fields missing; a failed connection terminates a user action.
+
+To proactively call a user’s attention to information that can help them avoid errors or undesirable outcomes, use a [warning](/components/callout#callout-variants). For example, telling a user an action might increase their bill; telling a user they’re about to hit a limit; telling a user a selection has notable limitations; telling a user an action could result in lost data.
+
+### Composition
+
+When developing an error message:
+
+<UnorderedList>
+  <ListItem>
+    <strong>Tell the user what happened, why it happened, and how to fix it (if they can).</strong>
+    <UnorderedList marginBottom="space0">
+      <ListItem>If you’re tight on space, prioritize what happened and how to fix it.</ListItem>
+      <ListItem>If you have extra room, explain what happens if the error isn’t fixed.</ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    <strong>Be as specific as possible.</strong> Develop a tailored error message for each possible cause.
+  </ListItem>
+  <ListItem>
+    <strong>Use plain, human-readable language.</strong> Prioritize active voice, but consider passive voice when the
+    emphasis should be on the outcome and not who/what caused the outcome.
+  </ListItem>
+  <ListItem>
+    <strong>Avoid blaming the user.</strong> Focus on what the user needs to do to move forward, not what they did not
+    do.
+  </ListItem>
+  <ListItem>
+    <strong>Avoid certain words.</strong> The language we use during an undesirable experience matters. Avoid the
+    following language:
+    <UnorderedList marginBottom="space0">
+      <ListItem>
+        <strong>Technical jargon.</strong> Don’t introduce terms your users may not understand.
+      </ListItem>
+      <ListItem>
+        <strong>Negative or alarmist words</strong> like: forbidden, illegal, prohibited, valid/invalid.
+      </ListItem>
+      <ListItem>
+        <strong>“Please.”</strong> It makes the action sound optional.
+      </ListItem>
+      <ListItem>
+        <strong>“Sorry.”</strong> Overusing it makes it lose its meaning. Reserve this for situations where something
+        highly detrimental has happened, like lost data.
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    <strong>Place the error message as close to the source of the error as possible.</strong>
+  </ListItem>
+</UnorderedList>
+
+### Accessibility
+
+When possible, use an icon and color to indicate an error. When using only an icon or color, ensure the content clearly indicates something went wrong.
+
+## Recommended phrasings and templates
+
+### Generic errors
+
+<Table tableLayout="fixed" striped>
```

Comment:
suggestion: I don't love Components in markdown, I get that sometimes we can't use markdown, but where ever you can we should just rely on the markdown equivilents. 

Main reason is they are all controlled centerally. So if we want to make changes to all content pages, we only need to do it once. With a lot of embedded components and html, we have to page by page to make site wide changes. 

There are definitely a few tables in here that can be plan markdown and a lot of the lists too.

Reply:
Updated lists, left some table for styling reasons